js-miniapp-sdk/src/miniapp.ts

```diff
@@ -203,4 +211,13 @@ export class MiniApp implements MiniAppFeatures, Ad, Platform {
   getPoints(): Promise<Points> {
     return getBridge().getPoints();
   }
+
+  getHostEnvironmentInfo(): Promise<HostEnvironmentInfo> {
+    return getBridge()
+      .getHostEnvironmentInfo()
+      .then(info => {
+        info.platform = HostPlatform[getBridge().platform.toUpperCase()];
```

Comment:
I think you don't need to add `toUpperCase`. That is making it so the value for `platform` doesn't match what's in the `Platform` type.

Probably the text should be either all uppercase or all lowercase, but I think it's too late to change it since we've already been exposing it through the `MiniApp.getPlatform` interface.

Reply:
Oh wait, sorry you can ignore my comment. I see now that this is uppercased just for getting the type via enum.

But maybe you could use something like this instead? `getBridge().platform as HostPlatform`

Reply:
@corycaywood somehow didn't work for me when testing at first but somehow does now so I guess that's better :D